sys/include/net/nanocoap.h

```diff
@@ -96,6 +96,12 @@
 #include <arpa/inet.h>
 #endif
 
+#if defined(MODULE_SOCK_UDP) || defined(DOXYGEN)
+#include "net/sock/udp.h"
+#else
+typedef void *sock_udp_ep_t;
```

Comment:
I think it really has to be: 
```suggestion
typedef void sock_udp_ep_t;
```

else `const sock_udp_ep_t *` would be `const void **`. :wink: